src/Microsoft.OpenApi/Interfaces/IOpenApiVersionService.cs

```diff
@@ -19,7 +19,7 @@ internal interface IOpenApiVersionService
         /// <param name="summary">The summary of the reference.</param>
         /// <param name="description">A reference description</param>
         /// <returns>The <see cref="OpenApiReference"/> object or null.</returns>
-        OpenApiReference ConvertToOpenApiReference(string reference, ReferenceType? type, string summary = null, string description = null);
+        OpenApiReference? ConvertToOpenApiReference(string reference, ReferenceType? type, string? summary = null, string? description = null);
```

Comment:
why do we accept a nullable reference type here? shouldn't it be required?

Reply:
It can be null in the case of an externally referenced file:
https://github.com/microsoft/OpenAPI.NET/blob/85c3a4e694e3410dd21e3a7b75b21dd1f7d24ad8/src/Microsoft.OpenApi/Reader/V2/OpenApiV2VersionService.cs#L142-L151